crates/autopilot/src/lib.rs

```diff
@@ -1,14 +1,3 @@
-pub mod arguments;
-pub mod database;
-pub mod decoded_settlement;
-pub mod driver_api;
-pub mod driver_model;
-pub mod event_updater;
-pub mod fok_limit_orders;
-pub mod on_settlement_event_updater;
-pub mod run_loop;
-pub mod solvable_orders;
-
 use {
```

Comment:
To me it makes more sense to keep module declarations above use imports.

Reply:
Me too... but `rust-analyzer` inserts `use statements; mod statements; code`, so I just fixed the order to LSP adding of use imports works as expected.

Otherwise, you end up with:

```rust
use new_path;

mod existing_mod;

use existing_path;
```

Reply:
I'm OK with either or, but it is inconsistent in the repo. I decided to just pick what my LSP server prefers 🤷.

Reply:
I wonder if this a rust analyzer setting. It would be better to change that if it exists.